edi_project_oca/models/project_project.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _name = "project.project"
+    _inherit = ["project.project", "edi.exchange.consumer.mixin"]
+
+    edi_disable_auto = fields.Boolean()
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        projects = super().create(vals_list)
+        for project, vals in zip(projects, vals_list, strict=True):
+            project._event("on_project_create").notify(project, vals)
```

Comment:
no need to trigger these events. They are already available by default w/ component_event as `on_record_create` and `on_record_write`. 
What's the final goal for you?